src/transformStateWithClones.js

```diff
@@ -6,8 +6,30 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
-  // write code here
-}
+  const finMass = [];
+  let someObj = { ...state };
+
+  for (const ch in actions) {
```

Comment:
what is the 'ch'? please named it relevant to understand what iterated over your loop
fix it everywhere please